src/course-about/course-intro/constants.ts

```diff
@@ -0,0 +1,13 @@
+import { Info as InfoIcon, CheckCircle as CheckCircleIcon } from '@openedx/paragon/icons';
+
+export const ALERT_VARIANTS = {
+  SUCCESS: 'success',
+  INFO: 'info',
+  DANGER: 'danger',
+} as const;
+
+export const ALERT_ICONS = {
+  [ALERT_VARIANTS.SUCCESS]: CheckCircleIcon,
+  [ALERT_VARIANTS.INFO]: InfoIcon,
+  [ALERT_VARIANTS.DANGER]: InfoIcon,
```

Comment:
Both are the info icon? It's not clear to me which design in the figma document is the danger example.